instrumentation/jaxrs-client/jaxrs-client-2.0/src/test/groovy/JaxRsClientTest.groovy

```diff
@@ -47,6 +52,44 @@ abstract class JaxRsClientTest extends HttpClientTest {
   }
 
   abstract ClientBuilder builder()
+
+  @Unroll
+  def "should properly convert HTTP status #statusCode to span error status"() {
```

Comment:
Do you mind if I create a separate issue or PR for this? Adding tests in `HttpClientTest` will affect all HTTP clients that we instrument and this may involve lots of changes that are unrelated to this PS's topic (RESTEasy).